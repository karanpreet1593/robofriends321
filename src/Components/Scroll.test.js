import React from 'react';
import { shallow } from 'enzyme';
import Scroll from './Scroll';

it('renders without crashing', () => {
  expect(shallow(<Scroll/>)).toMatchSnapshot();
});