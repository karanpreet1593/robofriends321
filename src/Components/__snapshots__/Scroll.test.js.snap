// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  style={
    Object {
      "border": "5px solid black",
      "height": "800px",
      "overflow": "Scroll",
    }
  }
/>
`;
